video_prediction/prediction_train.py

```diff
@@ -150,30 +155,45 @@ def __init__(self,
 
     # L2 loss, PSNR for eval.
     loss, psnr_all = 0.0, 0.0
+    try:
+        tf.equal(prefix,'train').eval()
+    except:
+        prefixs='initial'
+    else:
+        if tf.equal(prefix,'train').eval():
+            prefixs='train'
+        else:
+            prefixs='val'
```

Comment:
I'm testing the newest code, the last version tf.summary.scalar() must get a string a input, a tf.sensor will make error, so I add these code to make the output format same with the old version.